src/target_intacct/statistical_journal.py

```diff
@@ -141,18 +139,18 @@ def build_lines(
     journal_entries = []
     errored = False
 
-    # Create line items
-    line_item_types = ["Capacity", "BudgetedBillable"]
+    # Create list of account data the journal will contain
+    account_number_columns = [column for column in data.columns if column.startswith("AccountNo")]
```

Comment:
After this add a check that len(account_number_columns) > 0; if not then they didn't give us data in the right format and we can error.

Also after this let's validate that these are Intacct valid column names.

Reply:
@hulet The additional changes we discussed yesterday have been added, as well as functionality for custom Batch Titles and TR_Type (show the journal as an increase or decrease) and to allow for any of the standard Intacct dimensions (required fields to upload, ex: we require an employeeid to be associated with the capacity field). This enables use of any statistical account regardless of constraints, for example  in testing I used the PTO account which requires a project 
![image](https://github.com/FreshConsulting/target-intacct/assets/129103234/b99b9eef-148f-49dd-981d-5365515b4de5)